pages/common/sk.md

```diff
@@ -0,0 +1,21 @@
+# sk
+
+> Skim is a fuzzy finder written in rust.
```

Comment:
I'd write like this:
```suggestion
> Fuzzy finder written in rust.
```

Reply:
Shouldn't that be 
```suggestion
> Fuzzy finder written in Rust.
```

Reply:
Yes, `rust` instead of `Rust` was a mistake :)